modules/oke/cluster.tf

```diff
@@ -60,7 +63,13 @@ resource "oci_containerengine_cluster" "k8s_cluster" {
       services_cidr = var.cluster_options_kubernetes_network_config_services_cidr
     }
 
+    persistent_volume_config {
+      # defined_tags  = { "Operations.CostCenter" = "42" }
+      freeform_tags = var.freeform_tags["persistent_volume"]
+    }
+
     service_lb_config {
+      # defined_tags  = { "Operations.CostCenter" = "42" }
```

Comment:
Including commented-out code is generally frowned upon. I would suggest moving this to another example that demonstrates how to use defined tags.